html/comparison.html.haml

```diff
@@ -115,7 +115,7 @@
                   Changed
                   = ": {{diff.packages.changed.length}} packages"
                 %span.summary-part{"ng-show" => "diff.packages.common"}
-                  %a{:class=>"show-common-elements", href: "#packages_both"}
+                  %a{:class=>"show-common-elements", href: "#packages_both"}<
```

Comment:
Avoid defining `class` in attributes hash for static class names<br>Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace